modules/src/Components/NewTodo.tsx

```diff
@@ -26,60 +39,70 @@ const NewTodo = () => {
     return `${year}-${month}-${day}`;
   };
 
-  const submitHandler = async (e: FormEvent<HTMLFormElement>) => {
+  const handleSubmit = async (
+    values: ValuesType,
+    navigate: NavigateFunction
```

Comment:
Why navigate prop is required?

Reply:
Since, want to navigate to the home page after the submit button is pressed